File: LeetCode/Baseball Game/practice.py
def calPoints(operations):
    stack = []
    sum = 0
    for i in range(len(operations)):
        if operations[i] == "+":
            stack.append(stack[-1] + stack[-2])
        elif operations[i] == "D":
            stack.append(stack[-1] * 2)
        elif (operations[i] == "C"):
            stack.pop()
        else:
            stack.append(int(operations[i]))
    print(stack)
    for num in stack:
        sum = sum + num
    return sum

calPoints(["5","2","C","D","+"])

File: LeetCode/Baseball Game/practice.js
function calPoints(operations){
    let newOperations = []
    let x = 0
    for (let i = 0; i < operations.length; i ++){
        if (operations[i] == "+"){
            newOperations.push(newOperations[newOperations.length - 1] + newOperations[newOperations.length - 2])
        }
        else if (operations[i] == "D"){
            newOperations.push(newOperations[newOperations.length - 1] * 2)
        }
        else if (operations[i] == "C") {
            newOperations.pop()
        }
        else {
            newOperations.push(parseInt(operations[i]))
        }
    }
    console.log(newOperations)
    for (let j = 0; j < newOperations.length; j++){
        x += newOperations[j]
    }
    return x
}

console.log(calPoints(["5","2","C","D","+"]))